tools/debugger/debugger_tester.sh

```diff
@@ -8,7 +8,11 @@ RESULT_TEMP=`mktemp ${TEST_CASE}.out.XXXXXXXXXX`
 if [[ $IS_CLIENT_SOURCE == 0  ]] && [[ $TEST_CASE != *"tools/debugger/tests/client_source_multiple"* ]]; then
   ${ESCARGOT} --start-debug-server ${TEST_CASE}.js &
   sleep 1s
-  (cat "${TEST_CASE}.cmd" | ${DEBUGGER_CLIENT} --non-interactive) >${RESULT_TEMP} 2>&1
+  if [[ $TEST_CASE == *"tools/debugger/tests/do_command"* ]]; then
+    (echo -n -e "\x0a" | ${DEBUGGER_CLIENT} --non-interactive --command "e i;b do_command.js:2;c;c;c") >${RESULT_TEMP} 2>&1
+  else
```

Comment:
It seems that this `command` test actually does not run in CI.
Please check the test runner (`tools/run-tests.py`)